src/modules/auth/auth.controller.ts

```diff
@@ -58,4 +60,24 @@ export default class RegistrationController {
     const { id: user_id } = request['user'];
     return this.authService.enable2FA(user_id, password);
   }
+
+  @skipAuth()
+  @Post('signin-token')
+  @HttpCode(200)
+  @ApiOperation({ summary: 'Request Signin Token' })
+  @ApiResponse({ status: 200, description: 'Sign-in token sent to email', type: RequestSigninTokenDto })
+  @ApiResponse({ status: 400, description: 'Bad request' })
+  public async signInToken(@Body() body: RequestSigninTokenDto) {
+    return await this.authService.requestSignInToken(body);
+  }
+
+  @skipAuth()
```

Comment:
I believe this routes are protected if yes, remove the skipAuth decorator, the decorator is for bypassing authorization. By default all routes are protected

Reply:
fixed